cfme/tests/infrastructure/test_snapshot.py

```diff
@@ -573,7 +573,6 @@ def test_sui_create_snapshot():
         caseimportance: medium
         initialEstimate: 1/4h
         startsin: 5.9
-        title: SUI: Create snapshot from vm details page, snapshot page
                and service details page
```

Comment:
Need to delete this line also.

Reply:
nice catch

Reply:
indeed nice catch! I thought flake had caught me on all of these, but I guess this one didn't trigger?